cover-checker-console/src/main/java/com/naver/nid/cover/util/ParameterParser.java

```diff
@@ -48,17 +50,30 @@ public Parameter getParam(String... commandArgs) {
 			return null;
 		}
 
+		final List<String> coveragePaths = Arrays.asList(cmd.getOptionValues("c"));
+		final String[] coverageTypesArray = cmd.getOptionValues(COVERAGE_TYPE_OPTION);
+		List<String> coverageTypes = Arrays.asList(
+				coverageTypesArray == null
+						? new String[]{ "jacoco" }
+						: coverageTypesArray);
+
+		if ((coverageTypes.size() == 1) && coveragePaths.size() > 1) {
```

Comment:
what if 
```
coveragePaths.size() > 1 
&& coverageTypes.size() > 1
&& coverageTypes.size() < coveragePaths.size()
```
?
Smells like an NPE down the road.

Reply:
Yes, I've added validation for this now.